src/test/java/frc/robot/commands/VisionCommandBaseTest.java

```diff
@@ -0,0 +1,91 @@
+package frc.robot.commands;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import frc.robot.Constants;
+import frc.robot.networktables.DoubleEntryValue;
+import frc.robot.subsystems.ILimelightSubsystem;
+
+@RunWith(MockitoJUnitRunner.class)
+public class VisionCommandBaseTest {
+
+  @Mock
+  private ILimelightSubsystem limelightHigh;
+
+  @Mock
+  private ILimelightSubsystem limelightLow;
+
+  private VisionCommandBase getCommand() {
+    return new VisionCommandBase(limelightHigh, limelightLow);
```

Comment:
Nothing wrong here, but...
There is an annotation for creating the class under test and injecting mocks, @InjectMocks
If you don't use the annotation, the usual way of doing this would be to use the @Before annotation on a method and construct the object and save it in a field.